src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -431,6 +438,83 @@ export class SavedObjectsRepository {
     };
   }
 
+  /**
+   * Check what conflicts will result when creating a given array of saved objects. This includes "unresolvable conflicts", which are
+   * multi-namespace objects that exist in a different namespace; such conflicts cannot be resolved/overwritten.
+   */
+  async checkConflicts(
+    objects: SavedObjectsCheckConflictsObject[] = [],
+    options: SavedObjectsBaseOptions = {}
+  ): Promise<SavedObjectsCheckConflictsResponse> {
+    if (objects.length === 0) {
+      return { errors: [] };
+    }
+
+    const { namespace } = options;
+
+    let bulkGetRequestIndexCounter = 0;
+    const expectedBulkGetResults: Either[] = objects.map((object) => {
+      const { type, id } = object;
+
+      if (!this._allowedTypes.includes(type)) {
+        return {
+          tag: 'Left' as 'Left',
+          error: {
+            id,
+            type,
+            error: SavedObjectsErrorHelpers.createUnsupportedTypeError(type).output.payload,
```

Comment:
NIT: I see this `SavedObjectsErrorHelpers.createXXXError(stuff).output.payload` a lot. Maybe a `errorContent(err: Boom) => err.output.payload` util would makes sense?

Reply:
I thought about making a utility function, but it doesn't make the code more concise. I'll do it anyway for consistency's sake.